src/provisioner/cluster.py

```diff
@@ -110,13 +117,18 @@ def _validate(self, s_set: dict):
         if s_set_name is None:
             raise CortxProvisionerError(
                 errno.EINVAL, 'Missing name for the storage_set entry')
+
+        PLog.debug("Validating storage set '%s' properties" % s_set_name)
+
         required_keys_for_storage_set = ['durability', 'nodes']
         for k in required_keys_for_storage_set:
             if s_set.get(k) is None:
                 raise VError(
                     errno.EINVAL,
                     f"'{k}' property is unspecified for storage_set {s_set_name}.")
 
+        PLog.debug('Successfully validated storage set properties')
```

Comment:
Need to avoid messages which do not have a variable. If storage set validation fails then it would raise error or it can be assume to be validated.

Reply:
Removed it.

Reply:
Done.